ui/pages/create-account/connect-hardware/index.js

```diff
@@ -253,21 +253,23 @@ class ConnectHardwareForm extends Component {
       .then((_) => {
         this.context.trackEvent({
           category: EVENT.CATEGORIES.ACCOUNTS,
-          event: `Connected Account with: ${device}`,
+          event: EVENT_NAMES.ACCOUNT_IMPORT_COMPLETED,
           properties: {
-            action: 'Connected Hardware Wallet',
-            legacy_event: true,
+            account_type: EVENT.ACCOUNT_TYPES.HARDWARE,
+            account_hardware_type: device,
+            // account_hardware_version: null,
```

Comment:
Let's remove this commented code? Unless we want to implement this soon.

Reply:
i left it in so if/when an engineer wants to add hardware version, they'll already have the format.  i can add a note like this or remove it if suggested.

Reply:
I would probably remove it and have the `account_hardware_version` key in some user story for adding more detailed metrics across events.